### DISCUSSION TITLE
Next Release? (v3.9.x)

### DISCUSSION THREAD

**Discussion by silentpete** [getsops/sops#1444] [General]
Hello,

I keep reading where things have been fixed and will go into the next release...

I looked at the previous 4 releases and it seems like we're further out than most releases, in the sense of time...

Do we know when 3.9.x will get released?

**Comment by vladimirfx** [comment]
Very waiting `--mac-only-encrypted` because it principally changes the configuration management process using K8s secrets. How such a binary can be obtained?  (maybe there are public nightly builds)

**Reply by felixfontein** [reply]
If you have Golang, you can clone the repository and run `make install`, it will compile the binary and place it in `~/go/bin/` (assuming you didn't change your Go path). If it complains about vendoring things, run `make vendor`. That's the easiest way I know of compiliing it yourself.

If you simply want to download something, you can look at the latest build in https://github.com/getsops/sops/actions/workflows/cli.yml?query=branch%3Amain and download an artifact.

**Reply by vladimirfx** [reply]
Thank you

**Comment by felixfontein** [comment]
3.9.0 is out now: https://github.com/getsops/sops/releases/tag/v3.9.0

**Comment by silentpete** [comment]
I was able to review the merge and see how I could add `aws_profile:` to the `creation_rules:` and now if we create a file, it correctly puts the aws profile in place. No need for manual setting after the file is created.

Awesome, tysm!!
